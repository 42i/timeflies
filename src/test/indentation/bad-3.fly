wp aa
    xxx
        xxxx
        yyyy
      	zzzz
	bbb
    ccc
