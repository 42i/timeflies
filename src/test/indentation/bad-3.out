indentation/bad-3.fly:5: WARNING : indentation contains both spaces and tabs
indentation/bad-3.fly:5: ERROR : work package indentation error
indentation/bad-3.fly:6: ERROR : work package indentation error
